code-challenges.js

```diff
@@ -1,51 +1,45 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions
-
 // --------------------1) Create a function that takes in an array. Each time the function is run, remove the first item from the array and shuffle the remaining content. If the array is empty, return "The array is empty."
-
 var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example1 (can be a different order): ["yellow", "blue", "pink", "green"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
-
-
-
-
-
+//
+const minus1 = (array) => {
+  var collections = array.shift(0)
+  for(let i = 0; i < 0; i++){
+    return array
+  }
+}
+console.log(minus1(collections))
 // --------------------2) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
-
 var cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
-
-
-
-
-
+const cubed = (array) => {
+  let cubed = cubeAndSum1.map(value =>
+     math.cbrt()
+  )
+}
+cosole.log(cubed(cubeAndSum1))
 // --------------------3) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
-
 var nums1 = [3, 56, 90, -8, 0, 23, 6]
 // Expected output: [-8, 90]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
-
-
-
-
-
+//
+const minAndMax = (array) => {
+  array.filter(value => {
+    if value
+  })
+}
```

Comment:
I don't see a console log for this solution and it looks like your if statement is incomplete.